docs: Update Monitors add-signal steps to current UI (#2859)
wandb/docs: weave/guides/evaluation/monitors.mdx

Adding a signal turns on automated scoring so that Weave evaluates new production traces against that signal's criteria. To enable signals:

1. Navigate to [wandb.ai](https://wandb.ai/) and then open your Weave project.
1. In the Weave project sidebar, select **Monitors**.

> 1. To add a signal, click on its card to activate its checkbox, then click **Setup monitors**.

Author: ```suggestion
1. To add a signal to a project with no Monitors enabled, click on its card to activate its checkbox, then click **Setup monitors**.
```

> 1. To add a single signal, hover over its card and select the **Add signal** button. The signal starts scoring new traces immediately.

Author: ```suggestion
1. To add a signal, click on its card to activate its checkbox, then click **Setup monitors**.
```

> 1. To add several signals at once, select **Browse signals** in the **Recommended signals** header. This opens the **Add signals** drawer, which lists available signals grouped by category (such as Quality classifiers and Error classifiers), each with a checkbox. You can select individual signals, use **Enable all** for a group, or select **Create custom signal**.

Author: ```suggestion
1. To add signals to an existing Monitor, select **Browse signals** at the top-right on the **Monitors** page. This opens the **Add signals** drawer, which lists available signals grouped by category (such as Quality classifiers and Error classifiers), each with a checkbox. You can select individual signals, use **Enable all** for a group, or select **Create custom signal**.
```

> 1. Select the signals you want to turn on, then select **Add signals** in the drawer footer to enable them.

Author: ```suggestion
```

Adding a signal turns on automated scoring so that Weave evaluates new production traces against that signal's criteria. To enable signals:

1. Navigate to [wandb.ai](https://wandb.ai/) and then open your Weave project.
1. In the Weave project sidebar, select **Monitors**.

> 1. At the top of the Monitors page, find the **Recommended signals** section. Each card shows the signal name and a description.

Author: ```suggestion
```

1. To add a single signal, hover over its card and select the **Add signal** button. The signal starts scoring new traces immediately.

> 1. To add several signals at once, select **Browse signals** in the **Recommended signals** header. This opens the **Add signals** drawer, which lists available signals grouped by category, such as Quality and Error, each with a checkbox. You can select individual signals, use **Enable all** for a group, or select **Create custom signal**.

Author: ⚠️ **Reframed** — the grouping is real, but the drawer's rendered group headers are the literals **`Quality classifiers`** and **`Error classifiers`** (from `group.name`), not bare "Quality" and "Error".

```suggestion
1. To add several signals at once, select **Browse signals** in the **Recommended signals** header. This opens the **Add signals** drawer, which lists available signals grouped by category (such as Quality classifiers and Error classifiers), each with a checkbox. You can select individual signals, use **Enable all** for a group, or select **Create custom signal**.
```

**Source says:** `DEFAULT_CLASSIFIER_GROUPS` defines `name: 'Quality classifiers'` and `name: 'Error classifiers'`, and the drawer renders `{groupName}` = `group.name`. · **Source:** [core:defaultClassifiersConfig.ts#L421-L444](https://github.com/wandb/core/blob/ec797fd9b4cfea78a29229dc9dfbe99b5d582ce7/frontends/weave/src/components/PagePanelComponents/Home/Browse3/pages/MonitorsPage/defaultClassifiersConfig.ts#L421-L444)

_🤖 CoreWeave Docs Team's Validator — reframed_
<!-- cw-docs-validator -->

> 1. Select the signals you want to turn on, then select **Add signals** in the drawer footer to enable them.

Author: 🟡 **Partially confirmed** — the drawer footer button reads **`Add signals`** only when nothing is selected (`changeCount === 0`). Once the user selects signals (which this same step instructs), the label changes to **`Add N signals`** / `Add 1 signal`.

**Recommended:** soften the label reference so it isn't wrong after selection, e.g. "select the button in the drawer footer (labeled **Add N signals**) to enable them." Verify live: [wandb.ai](https://wandb.ai/) → Weave project → **Monitors** → **Browse signals**, select two signals, confirm the footer reads "Add 2 signals." · **Source:** [core:ClassifierCreateDrawer.tsx#L162-L173](https://github.com/wandb/core/blob/ec797fd9b4cfea78a29229dc9dfbe99b5d582ce7/frontends/weave/src/components/PagePanelComponents/Home/Browse3/pages/MonitorsPage/ClassifierCreateDrawer.tsx#L162-L173)

_🤖 CoreWeave Docs Team's Validator — partially confirmed_
<!-- cw-docs-validator -->

Adding a signal turns on automated scoring so that Weave evaluates new production traces against that signal's criteria. To enable signals:

1. Navigate to [wandb.ai](https://wandb.ai/) and then open your Weave project.
1. In the Weave project sidebar, select **Monitors**.
1. To add a signal to a project with no Monitors enabled, click on its card to activate its checkbox, then click **Setup monitors**.

> 1. To add signals to an existing Monitor, select **Browse signals** at the top-right on the **Monitors** page. This opens the **Add signals** drawer, which lists available signals grouped by category (such as Quality classifiers and Error classifiers), each with a checkbox. You can select individual signals, use **Enable all** for a group, or select **Create custom signal**.

Author: ```suggestion
1. To add signals to an existing Monitor, select **Browse signals** at the top-right on the **Monitors** page. This opens the **Add signals** drawer, which lists available signals grouped by category (such as Quality classifiers and Error classifiers), each with a checkbox. You can select individual signals, use **Enable all** for a group, or select **Create custom signal**. Then select **Add signals** at the bottom of the drawer. 
```
